applications/visualizer/frontend/src/components/viewers/ThreeD/SceneControls.tsx

```diff
@@ -77,7 +71,7 @@ function SceneControls({ cameraControlRef, isWireframe, setIsWireframe, recorder
   const stopRecording = async () => {
     if (recorderRef.current) {
       recorderRef.current.stopRecording({ type: "video/webm" });
-      recorderRef.current.download("CanvasRecording.webm", { type: "video/webm" });
+      recorderRef.current.download(workspace.name + ".webm", { type: "video/webm" });
```

Comment:
It's better to use string interpolation instead of concatenation, it's easier to read and maintain 

```js
`${workspace.name}.webm`
```